test/test_tokenizer.py

```diff
@@ -1132,6 +1132,12 @@ def test_correct_spaces() -> None:
     assert s == "Jón-sem var formaður—mótmælti málinu."
     s = t.correct_spaces("Það á   að geyma mjólkina við  20 ±  3 °C")
     assert s == "Það á að geyma mjólkina við 20±3° C"
+    s = t.correct_spaces("Við förum t.d. til Íslands o.s.frv.")
```

Comment:
Could add tests for cases that should not be considered abbreviations, for instance ordinal numbers squashed against letters, and wrong spacing:

"Ég fór til Írlands 6.júní og þar var 17.4°C hiti eða 230.3K."
"Þetta er setning.Þetta er önnur setning.Líka.En hvað með þetta?"

Reply:
Good point! Added those and handling for °C etc.